src/layers/XRLayer/XRLayer.js

```diff
@@ -71,6 +71,10 @@ export default class XRLayer extends Layer {
    * This function initializes the internal state.
    */
   initializeState() {
+    const { gl } = this.context;
+    // https://stackoverflow.com/questions/42789896/webgl-error-arraybuffer-not-big-enough-for-request-in-case-of-gl-luminance
```

Comment:
Can you add a small comment as well (leave the link). I don't know if I understand totally what an alignment is, but it sounds like can be certain that our data have an alignment of 1 (because they aren't interleaved RGB or RGBA) so we manually set this?

Reply:
Technically, I think we could use a value of 2 here for 16 bit data (sine rows will be 2 byte aligned) but I think that it is a bit of overkill to check the loader dtype here since 1 works fine.  Also, I don't want to muck around with resetting/checking this parameter every time data is loaded.

Reply:
I'm adding a comment into the code.